src/server/clickhouse/client.ts

```diff
@@ -62,7 +79,9 @@ export type ReactionType =
   | 'Question_Create'
   | 'Question_Delete'
   | 'Answer_Create'
-  | 'Answer_Delete';
+  | 'Answer_Delete'
+  | 'BountyEntry_Create'
+  | 'BountyEntry_Delete';
```

Comment:
Curious to me we don't wanna track bounty benefactor create/join?

Reply:
Same goes for awarding a bounty vs automatic awards (on expire)

Reply:
> Curious to me we don't wanna track bounty benefactor create/join?

Maybe it's more like I forgot to add it 😛 working on it now